server/services/withings/lib/withings.init.js

```diff
@@ -0,0 +1,347 @@
+const uuid = require('uuid');
+const OAuth2Manager = require('../../../lib/oauth2');
+const logger = require('../../../utils/logger');
+const {
+  DEVICE_FEATURE_CATEGORIES,
+  DEVICE_POLL_FREQUENCIES,
+  DEVICE_FEATURE_TYPES,
+  DEVICE_FEATURE_UNITS,
+} = require('../../../utils/constants');
+
+/**
+ * @description Build a new gladys device from withings device.
+ *
+ * @param {Object} withingsDevice - Withings device to transform.
+ * @param {string} serviceId - Withings service id.
+ * @returns {Object} Return a gladys device.
+ * @example
+ * withings.buildNewDevice('7fdsf4s68r4gfr68f4r63csd7f6f4c3r85', '78v4f3df83g74v1fsd8375f63gvrf5c');
+ */
+function buildNewDevice(withingsDevice, serviceId) {
+  // Build unique id for the device
+  const uniqueId = uuid.v4();
+
+  // Build params for all device
+  const newParams = [
+    {
+      name: 'withingsDeviceId',
```

Comment:
I think usually we save params in caps lock, so it would be here WITHINGS_DEVICE_ID

Reply:
I fix it